packages/microservice/src/bus-adapters/nats-bus.adapter.ts

```diff
@@ -0,0 +1,61 @@
+import { Logger } from '@nestjs/common';
+import { fromEvent, merge } from 'rxjs';
+import { map, first } from 'rxjs/operators';
+import { IBusAdapter } from '../interfaces/bus/bus-adapter.interface';
+import { IOnInit } from '../interfaces/lifecycles';
+import { ISetOptions } from '../interfaces/set-options.interface';
+import { ITransferData } from '../interfaces/transfer-data';
+import { TransferDataDto } from '../interfaces/transfer-data-dto.interface';
+import { loadPackage } from '../utils/load-package.util';
+
+let natsPackage: any = {};
```

Comment:
As many people going to use our adapter to avoid side effects please put this within the adapter class.

Reply:
or avoid making something like that whatever the context